### ISSUE TITLE
autoColumnSize doesn't work in Safari

### ISSUE THREAD

**Issue by JacobEvelyn** [handsontable/handsontable#3847]
### Description

Columns should be intelligently sized by default (or when explicitly using the `autoColumnSize` setting). This behavior works correctly in Chrome but does not in Safari.

### Steps to reproduce

1. Make a basic Handsontable with headers and cell data large enough to need `autoColumnSize` functionality.

Notice the difference between Chrome (correct):

<img width="578" alt="screen shot 2016-10-28 at 4 36 23 pm" src="https://cloud.githubusercontent.com/assets/1114569/19821812/6a3d3eca-9d2e-11e6-8a29-59fa6fea27d9.png">

and Safari (incorrect):

<img width="560" alt="screen shot 2016-10-28 at 4 36 08 pm" src="https://cloud.githubusercontent.com/assets/1114569/19821824/780c0144-9d2e-11e6-89dd-1500a5678096.png">

### Demo

[Fiddle here](http://jsfiddle.net/x9382xfc/).

### Your environment
* Version: 0.28.4/1.7.4
* Browser Name and version: Safari 10.0.1
* Operating System: OS X 10.11.6

**Comment by swistach** [comment]
@JacobEvelyn 
In 0.29.0 it seems to be ok.
Can you confirm that?

**Comment by JacobEvelyn** [comment]
@swistach no, it's still broken. The fiddle I posted still doesn't work on Safari 10.0.1, and it uses the latest version of Handsontable.

**Comment by AMBudnik** [comment]
@swistach unfrotunately it is still replicable, @budnix checked it on Safari 10. It works as expected on Safari 9
